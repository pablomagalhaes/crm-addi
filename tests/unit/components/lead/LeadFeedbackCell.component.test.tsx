import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import LeadFeedbackCell from '@/components/lead/LeadFeedbackCell.component';

const lead = {
  existsInRegisty: true,
  matchWithRegisty: true,
  hasJudicialRecord: true,
  score: 0,
};

describe('<LeadFeedbackCell />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<LeadFeedbackCell {...lead} />);
    expect(wrapper).toMatchSnapshot();
  });
});
