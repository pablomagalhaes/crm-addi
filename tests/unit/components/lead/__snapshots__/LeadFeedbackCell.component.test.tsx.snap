// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeadFeedbackCell /> should match snapchat 1`] = `
<Fragment>
  <Styled(styled.p)>
    - Has Judicial Record
  </Styled(styled.p)>
  <Styled(styled.p)>
    - Low score
  </Styled(styled.p)>
</Fragment>
`;
